tests/monotouch-test/Foundation/UbiquitousKeyValueStoreTest.cs

```diff
@@ -32,8 +32,6 @@ public class NSUbiquitousKeyValueStoreTest {
 		[Test]
 		public void Indexer ()
 		{
-			Assert.Ignore ("Doesn't work on watchOS");
```

Comment:
Looking at it, might have removed it because the whole test is inside a #if !__WATCHOS__ block, so this is just ignoring the test for other platforms.

Thoughts anyone?